Tools/Scripts/webkitpy/common/checkout/checkout.py

```diff
@@ -49,8 +49,8 @@
 # FIXME: Move a bunch of ChangeLog-specific processing from SCM to this object.
 # NOTE: All paths returned from this class should be absolute.
 class Checkout(object):
-    COMMIT_SUBJECT_RE = re.compile(b'Subject: \[PATCH ?(\d+\/\d+)?] (.+)')
-    FILTER_BRANCH_PROGRAM = '''import re
+    COMMIT_SUBJECT_RE = re.compile(b'Subject: \\[PATCH ?(\\d+\\/\\d+)?] (.+)')
```

Comment:
I'd rather just make this a raw string:

```suggestion
    COMMIT_SUBJECT_RE = re.compile(br'Subject: \[PATCH ?(\d+\/\d+)?] (.+)')
```

Reply:
Ah, nice. Didn't know we could use both `b` and `r` at the same time!

Reply:
Annoyingly, prior to Python 3.3, they _must_ be in that order (`br`)—because I always want a "raw byte string", not a "byte raw string"!